### ISSUE TITLE
Clarify the relationship between this GitHub integration and the Teams GitHub tab

### ISSUE THREAD

**Issue by glopesdev** [integrations/microsoft-teams#42]
It seems these two are related somehow (maybe even the same app), but the relationship between both is not very clear. Specifically, it would be fundamental to understand the relationship between the `subscribe` command and adding a tab pointing to the repository.

**Comment by glopesdev** [comment]
After some exploration, it is now clear that the bot and the tab are two separate applications, each with its own OAuth token that needs to be authorized and managed independently for each organization on GitHub. It would still be worth it to update the README file and/or blog posts to clarify this relationship, in my opinion.

**Comment by ashokirla** [comment]
@glopesdev, we dont have tabs support in our new GitHub (Preview) app built by GitHub. You might be referring to another old GitHub integration built by TCS.

**Comment by glopesdev** [comment]
@ashokirla yes, this is now clear to me. However, being a newcomer to Teams it was not clear these were separate at install time, as both integrations share the same icon either going through tabs or apps. I ended up installing both without realizing they were different things until I had the need to grant OAuth authentication for some private repos.

The TCS integration is still really useful to visually integrate GitHub issues as a tab on teams, which is invaluable for long-term project management.

Is tabs support planned (even if in the long-term roadmap) for this integration? That would be amazing, and likely in that moment the separation between both integrations would really need to be clarified.

For now feel free to close the issue if no action item is deemed necessary.

**Comment by ashokirla** [comment]
@glopesdev We are following up with the Teams ecosystem to rename the old TCS app to avoid confusion. We are currently working on GA features and we don't have plans to introduce Tabs as part of it. Post GA, we will evaluate the possibility of extending our support for Project management. 

Once again thanks for your support. Please keeping sharing your feedback to make our app better.

For us to track better, I am renaming this Issue to ask for 'Tabs and Project management'

**Comment by tomkerkhove** [comment]
We could like to have this as well since we are using more and more GitHub Issues, Milestones & Projects.
